vscode/src/explorer.ts

```diff
@@ -907,7 +907,7 @@ export function createTreeViewService(log : vscode.OutputChannel, c : NbLanguage
         let v = args as Visualizer;
         let ok = await c.sendRequest(NodeInfoRequest.destroy, { nodeId : v.data.id });
         if (!ok) {
-            vscode.window.showErrorMessage(l10n.value("jdk.explorer.error_message.cannotDeleteNode" , {
```

Comment:
thanks for testing @shivam71 , please rename this key in bundle you can keep this key as is here for clear context

Reply:
Done